src/actions/trackerActions.js

```diff
@@ -337,11 +256,20 @@ const makeSelectionAction = actionCreator => (...args: any[]) => (
   dispatch(clearSelection())
 }
 
+export { editTimestamp, createPoint }
+export const copySelectedWorklog = (...args: any[]) => (
+  dispatch: Dispatch,
+  getState: GetState
+) => {
+  const { points } = getState()
+  dispatch(makeSelectionAction(copyWorklog)(...args))
+  dispatch(setCursor(points.length - 1))
```

Comment:
I think the safest way is to write it as this:
```suggestion
  dispatch(setCursor(getState().points.length - 1))
```
because the copy worklog may insert one or not insert one (if nothing is selected nor hovered). also after https://github.com/vacuumlabs/fine-tracker/issues/354, it will be copying possibly multiple worklogs :)